app/views/candidate_interface/personal_details/immigration_status/_shared_form.html.erb

```diff
@@ -0,0 +1,28 @@
+<%= f.govuk_error_summary %>
+
+<% if @form.eu_nationality? %>
+  <%= f.govuk_radio_buttons_fieldset(
+    :immigration_status,
+    legend: { text: t('page_titles.immigration_status'), size: 'l' },
+  ) do %>
+    <%= f.govuk_radio_button :immigration_status, 'eu_settled', label: { text: 'EU settled status' }, link_errors: true %>
+    <%= f.govuk_radio_button :immigration_status, 'eu_pre_settled', label: { text: 'EU pre-settled status' } %>
+    <%= f.govuk_radio_button :immigration_status, 'other', label: { text: 'Other' } do %>
+      <%= f.govuk_text_area(
+        :immigration_status_details,
+        label: { text: 'Give any relevant information', class: 'govuk-label govuk-label--s' },
```

Comment:
Last minute copy-change, sorry, but can we change this to “Immigration status” as I think we only need the title of their immigration status at the point, rather than encouraging them to go into a lot of details...

```suggestion
        label: { text: 'Immigration status', class: 'govuk-label govuk-label--s' },
```